src/gameplay_v2/states/baseStates/discussion.ts

```diff
@@ -0,0 +1,6 @@
+import { IState } from '../state';
+
+export class VotingMission implements IState {
+  static state = 'Discussion';
+  state = 'Discussion';
```

Comment:
Let's also define all the states in some enum. This makes sense to enum.

Reply:
Enum has been created. Newer updates are upcoming soon :0